projects/ad9361/src/main.c

```diff
@@ -498,6 +502,15 @@ int main(void)
 		return status;
 	}
 	gpio_direction_output(default_init_param.gpio_desc_resetb, 1);
+
+	spi_param.chip_select = default_init_param.id;
+
+	status = spi_init(&default_init_param.spi, &spi_param);
+	if (status != SUCCESS) {
+		printf("spi_init() error: %"PRIi32"\n", status);
+		return status;
+	}
+
```

Comment:
I would split this commit in two, with the change to the headers in a separate commit.